http-server/src/main/java/xtdb/query/Basis.java

```diff
@@ -0,0 +1,6 @@
+package xtdb.query;
+
+import java.time.Instant;
+
+public record Basis(TransactionKey atTx, Instant currentTime){};
```

Comment:
this is great - I suspect this will become part of the Java API, so could it go in the `api` module instead? (and hence become a regular class, because Java 11)